tests/scripts/test_scripts.py

```diff
@@ -132,11 +105,43 @@ def test_main_console(script_mod):
 RL_SAC_NAMED_CONFIGS = ["rl.sac", "train.sac"]
 
 
-@pytest.mark.parametrize("config", PREFERENCE_COMPARISON_CONFIGS)
-def test_train_preference_comparisons_main(tmpdir, config):
+@pytest.fixture(
+    params=[
+        "plain",
+        "with_expert_trajectories",
+        "warmstart",
+        "with_checkpoints",
+    ],
+)
+def preference_comparison_config(request):
```

Comment:
nice idea to give the configs explanatory names!

Reply:
thx